inst/bench/bench.R

```diff
@@ -73,49 +72,45 @@ build_graph <- function(left = -Inf, right = Inf) {
   )
   po_fmean %>>%
     po_fvar %>>%
-    po_slope
+    po_fslope
 }
 
 analyse_fda <- function(graph, task) {
   graph$train(task)
 }
 
-benchmark_fda <- function(n_weeks = 52,
-                          n_patients = 100,
-                          window_start = 2,
-                          window_end = NULL,
-                          type = c("reg", "irreg"),
-                          seed = 1234,
-                          times = 100L) {
-  type <- match.arg(type)
-  window_end <- if (is.null(window_end)) floor(n_weeks * 0.8)
+results <- bench::press(
+  type = c("reg", "irreg"),
+  n_weeks = c(10, 52, 100),
+  n_patients = c(10, 100, 1000),
+  {
+    window_start <- 2
```

Comment:
Ich glaube diese windows sind nicht ganz optimal.

Reply:
Weil wir bei mlr3fda von links kucken fuer die linke grenze und von rechs fuer die rechte grenze werden diese positionen schnell gefunden.

Reply:
(weil die daten ja sortiert sind)

Reply:
Ich glaube aber nicht, dass es einen unterschied macht

Reply:
vllt sollten wir die "arg" spalte in den data tables / tibbles auch vor dem benchmark sortieren? (im reg fall ist das ja glaube ich eh der Fall, aber im irreg nicht)